AStar1.txt

```diff
@@ -8,7 +8,7 @@ def haversine(lon1, lat1, lon2, lat2):
     c = 2 * asin(sqrt(a)) 
     r = 6371
     return c * r
-
+#Lully
```

Comment:
@amoghwagh Why have you done something like this? It is atrocious.

Reply:
Screw you.